File: src/main/java/com/sequenceiq/cloudbreak/controller/validation/blueprint/StackServiceComponentDescriptors.java
package com.sequenceiq.cloudbreak.controller.validation.blueprint;

import java.util.Map;

public class StackServiceComponentDescriptors {
    private Map<String, StackServiceComponentDescriptor> stackServiceComponentDescriptorMap;

    public StackServiceComponentDescriptors(Map<String, StackServiceComponentDescriptor> stackServiceComponentDescriptorMap) {
        this.stackServiceComponentDescriptorMap = stackServiceComponentDescriptorMap;
    }

    public StackServiceComponentDescriptor get(String name) {
        return stackServiceComponentDescriptorMap.get(name);
    }
}

File: core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/blueprint/StackServiceComponentDescriptors.kt
package com.sequenceiq.cloudbreak.controller.validation.blueprint

class StackServiceComponentDescriptors(private val stackServiceComponentDescriptorMap: Map<String, StackServiceComponentDescriptor>) {

    operator fun get(name: String): StackServiceComponentDescriptor {
        return stackServiceComponentDescriptorMap[name]
    }
}
